modules/olmv1-installing-an-operator.adoc

```diff
@@ -29,15 +29,15 @@ spec:
   installNamespace: <namespace>
   serviceAccount:
     name: <service_account>
-  channel: <channel>
+  channels: [<channel>]
```

Comment:
@michaelryanpeter the fields of the example is not correct.
here is the current example.

```yaml
 apiVersion: olm.operatorframework.io/v1
  kind: ClusterExtension
  metadata:
    name: enough-75493
  spec:
    namespace: ns-75493
    serviceAccount:
      name: sa75493
    source:
      sourceType: Catalog
      catalog:
        packageName: nginx-ok-v3277775493
        channels:
          - alpha
        version: ">=0.0.1"
```

it seems you forget to update it because you have https://github.com/openshift/openshift-docs/pull/88848/files#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R87-R95

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/88848/files#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R23-R37